cogs5e/sheets/gsheet.py

```diff
@@ -407,11 +407,21 @@ def get_resistances(self):
             if resist:
                 out['resist'].append(resist.lower())
 
-        for immune_row in range(69, 80):  # AE69:AE79
-            immune = self.additional.value(f"AE{immune_row}")
+        for immune_row_old in range(69, 80):  # AE69:AE79
+            immune = self.additional.value(f"AE{immune_row_old}")
             if immune:
                 out['immune'].append(immune.lower())
 
+        for immune_row in range(69, 80):  # AB69:AB79
+            immune = self.additional.value(f"AB{immune_row}")
+            if immune:
+                out['immune'].append(immune.lower())
+
+        for vuln_row in range(69, 80):  # AI69:AI79
+            vuln = self.additional.value(f"AI{vuln_row}")
+            if vuln:
+                out['vuln'].append(immune.lower())
+
```

Comment:
Shouldn't this use an if statement checking the version numbers for the proper cells, rather than applying both?
(How it is could apply immunities multiple times, or redundantly iterate, not quite sure how the API handles pulling from merged cells.)

Reply:
Nevermind, checks only the main cell